frontend/e-buddy/src/app/components/ProgressBar.tsx

```
@@ -0,0 +1,20 @@
+
+interface ProgressBarProps {
+  progress: number;
+}
+
+const ProgressBar: React.FC<ProgressBarProps> = ({ progress }) => {
+  return (
+    <div className="w-full bg-gray-200 rounded-full h-3">
+      <div
+        className="bg-blue-500 h-3 rounded-full transition-all duration-300"
+        style={{ width: `${progress}%` }}
+      ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)